### ISSUE TITLE
[go] Something changed since v2017.02.07

### ISSUE THREAD

**Issue by StevenDTX** [rg3/youtube-dl#12210]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.02.21*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [X] I've **verified** and **I assure** that I'm running youtube-dl **2017.02.21**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [X] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [X] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Not working 2017.02.21:
```
E:\>youtube-dl.exe --download-archive D:\scripts\Auto-TV-Download\archive.txt --restrict-filenames --continue --no-check-certificate --verbose --ap-mso DTV --ap-username PRIVATE --ap-password PRIVATE -o American.Housewife.S01E08.Westport.Cotillion.mp4 https://abc.go.com/shows/american-housewife/episode-guide/season-1/08-westport-cotillion
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--download-archive', 'D:\\scripts\\Auto-TV-Download\\archive.txt', '--restrict-filenames', '--continue', '--no-check-certificate', '--verbose', '--ap-mso', 'DTV', '--ap-username', 'PRIVATE', '--ap-password', 'PRIVATE', '-o', 'American.Housewife.S01E08.Westport.Cotillion.mp4', 'https://abc.go.com/shows/american-housewife/episode-guide/season-1/08-westport-cotillion']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2017.02.21
[debug] Python version 3.4.4 - Windows-10-10.0.14393
[debug] exe versions: ffmpeg N-82966-g6993bb4, ffprobe N-72383-g7206b94, rtmpdump 2.4
[debug] Proxy map: {}
[debug] Using fake IP 3.193.109.66 (US) as X-Forwarded-For.
[Go] westport-cotillion: Downloading webpage
[Go] 3563115: Downloading JSON metadata
[Go] 3563115: Retrieving Media Token
[Go] 3563115: Downloading Provider Redirect Page
[Go] 3563115: Downloading Provider Login Page
[Go] 3563115: Logging in
[Go] 3563115: Confirming Login
[Go] 3563115: Retrieving Session
[Go] 3563115: Retrieving Authorization Token
[Go] 3563115: Downloading JSON metadata
ERROR: Go said: 6007:Unable to retrieve required files for playback. Please try again in a few minutes.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpqrnhzzwk\build\youtube_dl\YoutubeDL.py", line 704, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpqrnhzzwk\build\youtube_dl\extractor\common.py", line 427, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpqrnhzzwk\build\youtube_dl\extractor\go.py", line 110, in _real_extract
youtube_dl.utils.ExtractorError: Go said: 6007:Unable to retrieve required files for playback. Please try again in a few minutes.
```

Working 2017.02.07:
```
C:\support\Python36\lib\youtube-dl-master>c:\support\Python36\python.exe -m youtube-dl --download-archive D:\scripts\Auto-TV-Download\archive.txt --restrict-filenames --continue --no-check-certificate --verbose --ap-mso DTV --ap-username PRIVATE --ap-password PRIVATE -o American.Housewife.S01E08.Westport.Cotillion.mp4 https://abc.go.com/shows/american-housewife/episode-guide/season-1/08-westport-cotillion
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--download-archive', 'D:\\scripts\\Auto-TV-Download\\archive.txt', '--restrict-filenames', '--continue', '--no-check-certificate', '--verbose', '--ap-mso', 'DTV', '--ap-username', 'PRIVATE', '--ap-password', 'PRIVATE', '-o', 'American.Housewife.S01E08.Westport.Cotillion.mp4', 'https://abc.go.com/shows/american-housewife/episode-guide/season-1/08-westport-cotillion']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] youtube-dl version 2017.02.07
[debug] Python version 3.6.0b1 - Windows-10-10.0.14393-SP0
[debug] exe versions: ffmpeg N-82966-g6993bb4, ffprobe N-72383-g7206b94, rtmpdump 2.4
[debug] Proxy map: {}
[Go] westport-cotillion: Downloading webpage
[Go] 3563115: Downloading JSON metadata
[Go] 3563115: Downloading Provider Redirect Page
[Go] 3563115: Downloading Provider Login Page
[Go] 3563115: Logging in
[Go] 3563115: Confirming Login
[Go] 3563115: Retrieving Session
[Go] 3563115: Retrieving Authorization Token
[Go] 3563115: Retrieving Media Token
[Go] 3563115: Downloading JSON metadata
[Go] 3563115: Downloading m3u8 information
[download] Westport Cotillion has already been recorded in archive
```

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://abc.go.com/shows/american-housewife/episode-guide/season-1/08-westport-cotillion

---

### Description of your *issue*, suggested solution and other information

ERROR: Go said: 6007:Unable to retrieve required files for playback. Please try again in a few minutes.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpqrnhzzwk\build\youtube_dl\YoutubeDL.py", line 704, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpqrnhzzwk\build\youtube_dl\extractor\common.py", line 427, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpqrnhzzwk\build\youtube_dl\extractor\go.py", line 110, in _real_extract
youtube_dl.utils.ExtractorError: Go said: 6007:Unable to retrieve required files for playback. Please try again in a few minutes.

**Comment by RuneMagle** [comment]
Do you NEED Tv Provider login, or can you fix this without?

**Comment by yan12125** [comment]
@RuneMagle Apparently it's already fixed. If not please open a new issue and paste verbose logs.

**Comment by RuneMagle** [comment]
It's not fixed at all. I'll open a new issue now.

**Comment by StevenDTX** [comment]
Works just fine.  It was fixed in v2017.02.24.

```

E:\>youtube-dl.exe --verbose https://abc.go.com/shows/american-housewife/episode-guide/season-1/08-westport-cotillion
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', 'https://abc.go.com/shows/american-housewife/episode-guide/season-1/08-westport-cotillion']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2017.03.02
[debug] Python version 3.4.4 - Windows-10-10.0.14393
[debug] exe versions: ffmpeg N-82966-g6993bb4, ffprobe N-72383-g7206b94, rtmpdump 2.4
[debug] Proxy map: {}
[Go] westport-cotillion: Downloading webpage
[Go] 3563115: Downloading JSON metadata
[Go] 3563115: Retrieving Authorization Token
[Go] 3563115: Retrieving Media Token
[Go] 3563115: Downloading JSON metadata
[Go] 3563115: Downloading m3u8 information
[debug] Invoking downloader on 'http://content-auso1.uplynk.com/8d39430b70fe487e94e8f62e3c9a104c/j.m3u8?exp=1488651116&ct=a&oid=d874124ecca24c88a3c9575e78686acf&eid=10081208&iph=d6f86aa9c907b9fd9e516da8fededd6f334288018690a5fb7968b452b27fa498&rays=jihgfedcb&euid=7F1A2790-AD56-4197-96EF-7CF36A3EF9D8_000_1_001_lf_01-06-00_NA&cdn=ec&stgcfg=datg&pp2ip=0&sig=3e2ae51357903f80305ff0cd485b4b8f5e6c5d4dd24abe60f694bb2da79c5425&pbs=691716254a024fc3bdc68ec7d830cd14'
[download] Destination: Westport Cotillion-3563115.mp4
[debug] ffmpeg command line: ffmpeg -y -headers 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:10.0) Gecko/20150101 Firefox/47.0 (Chrome)
Accept-Language: en-us,en;q=0.5
Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Encoding: gzip, deflate
' -i 'http://content-auso1.uplynk.com/8d39430b70fe487e94e8f62e3c9a104c/j.m3u8?exp=1488651116&ct=a&oid=d874124ecca24c88a3c9575e78686acf&eid=10081208&iph=d6f86aa9c907b9fd9e516da8fededd6f334288018690a5fb7968b452b27fa498&rays=jihgfedcb&euid=7F1A2790-AD56-4197-96EF-7CF36A3EF9D8_000_1_001_lf_01-06-00_NA&cdn=ec&stgcfg=datg&pp2ip=0&sig=3e2ae51357903f80305ff0cd485b4b8f5e6c5d4dd24abe60f694bb2da79c5425&pbs=691716254a024fc3bdc68ec7d830cd14' -c copy -f mp4 -bsf:a aac_adtstoasc 'file:Westport Cotillion-3563115.mp4.part'
ffmpeg version N-82966-g6993bb4 Copyright (c) 2000-2016 the FFmpeg developers
  built with gcc 5.4.0 (GCC)
  configuration: --enable-gpl --enable-version3 --enable-dxva2 --enable-libmfx --enable-nvenc --enable-avisynth --enable-bzlib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenh264 --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzimg --enable-lzma --enable-decklink --enable-zlib
  libavutil      55. 43.100 / 55. 43.100
  libavcodec     57. 70.100 / 57. 70.100
  libavformat    57. 61.100 / 57. 61.100
  libavdevice    57.  2.100 / 57.  2.100
  libavfilter     6. 68.100 /  6. 68.100
  libswscale      4.  3.101 /  4.  3.101
  libswresample   2.  4.100 /  2.  4.100
  libpostproc    54.  2.100 / 54.  2.100
[aac @ 0000000002775940] element type mismatch 1 != 0
Input #0, hls,applehttp, from 'http://content-auso1.uplynk.com/8d39430b70fe487e94e8f62e3c9a104c/j.m3u8?exp=1488651116&ct=a&oid=d874124ecca24c88a3c9575e78686acf&eid=10081208&iph=d6f86aa9c907b9fd9e516da8fededd6f334288018690a5fb7968b452b27fa498&rays=jihgfedcb&euid=7F1A2790-AD56-4197-96EF-7CF36A3EF9D8_000_1_001_lf_01-06-00_NA&cdn=ec&stgcfg=datg&pp2ip=0&sig=3e2ae51357903f80305ff0cd485b4b8f5e6c5d4dd24abe60f694bb2da79c5425&pbs=691716254a024fc3bdc68ec7d830cd14':
  Duration: 00:20:24.94, start: 0.000000, bitrate: 0 kb/s
  Program 0
    Metadata:
      variant_bitrate : 0
    Stream #0:0: Audio: aac (HE-AAC) ([15][0][0][0] / 0x000F), 48000 Hz, stereo, fltp
    Metadata:
      variant_bitrate : 0
    Stream #0:1: Video: h264 (High) ([27][0][0][0] / 0x001B), yuv420p, 1280x720, 23.98 tbr, 90k tbn, 48 tbc
    Metadata:
      variant_bitrate : 0
    Stream #0:2: Data: timed_id3 (ID3  / 0x20334449)
    Metadata:
      variant_bitrate : 0
Output #0, mp4, to 'file:Westport Cotillion-3563115.mp4.part':
  Metadata:
    encoder         : Lavf57.61.100
    Stream #0:0: Video: h264 (High) ([33][0][0][0] / 0x0021), yuv420p, 1280x720, q=2-31, 23.98 tbr, 90k tbn, 90k tbc
    Metadata:
      variant_bitrate : 0
    Stream #0:1: Audio: aac (HE-AAC) ([64][0][0][0] / 0x0040), 48000 Hz, stereo, fltp
    Metadata:
      variant_bitrate : 0
Stream mapping:
  Stream #0:1 -> #0:0 (copy)
  Stream #0:0 -> #0:1 (copy)
Press [q] to stop, [?] for help
frame=  659 fps=338 q=-1.0 Lsize=   11568kB time=00:00:27.64 bitrate=3427.7kbits/s speed=14.2x
video:11134kB audio:429kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.046797%
Exiting normally, received signal 2.

ERROR: Interrupted by user

E:\>

```
